lib/ueberauth/strategy/auth0.ex

```diff
@@ -9,17 +9,17 @@ defmodule Ueberauth.Strategy.Auth0 do
         providers: [
           auth0: { Ueberauth.Strategy.Auth0, [uid_field: :email] }
         ]
-  Default is `:user_id`
+  Default is `:sub`
 
   To set the default ['scopes'](https://auth0.com/docs/scopes) (permissions):
       config :ueberauth, Ueberauth,
         providers: [
-          auth0: { Ueberauth.Strategy.Auth0, [default_scope: "openid email"] }
+          auth0: { Ueberauth.Strategy.Auth0, [default_scope: "openid profile email"] }
         ]
-  Deafult is "openid email"
+  Deafult is `"openid profile email"`
   """
-  use Ueberauth.Strategy, uid_field: :user_id,
-                          default_scope: "openid email",
+  use Ueberauth.Strategy, uid_field: :sub,
+                          default_scope: "openid profile email",
```

Comment:
Since `profile` is not a common scope for other Überauth strategies, I would prefer not to include it. The user can opt-in if need.

Reply:
Ok, then we should improve the docs:

1. what are the possible scopes? link: https://auth0.com/docs/scopes/current
2. what are the default scopes in our lib

Reply:
@sntran

Reply:
Thanks for digging into this!